docs/user-guide/building-isos.rst

```diff
@@ -79,6 +79,23 @@ You have to provide the following parameters:
 * ``--exclude-dns``: Flag to add the nameservers (and other DNS information) to the append line or not. This only has
   an effect in case you supply ``--systems`` and the system contains the ``--name-servers`` configuration.
 
+Building ISOs for Secure Boot
+#############################
```

Comment:
I would love if you add the `--esp` parameter to the above common parameters list.

Reply:
+1, done :)